### PR TITLE
Favorite heroes

### PR DESCRIPTION
**Pull request by synap5e**
This adds support for fetching favorite heroes from the database.
`create_default_user.py` sets the default favorite heroes which can be changed with OnSetFavoriteHero.

DbfCard has also had the CardClass field added (I think this is what you wanted @jleclanche). This field is used by create_default_user for assigning the default heroes to the correct class, but this could be done another way (`DEFAULT_FAVORITE_HEROES` holding 2-tuples of class, hero) if this isn't what you wanted.

### CODE REVIEW

**Comment by synap5e** [comment]
- [x] Please use double quotes here
- [x] Call this `values`
- [x] Do you really need that? just iterate over the `dbf_hero` table. You can do a join with the `dbf_card `table.

**Comment by synap5e** [comment]
Or should I do `SELECT dbf_card.id, dbf_card.card_class FROM dbf_card JOIN dbf_hero ON dbf_card.note_mini_guid = dbf_hero.card_id WHERE not dbf_hero.store_bought` ?

**Comment by jleclanche** [comment]
Whatever works.

**Comment by synap5e** [comment]
All done.

File: pegasus/db.go [hunk 1]
```diff
@@ -87,6 +95,7 @@ type DbfCard struct {
 	NoteMiniGuid  string
 	IsCollectible bool
 	NameEnus      string
+	CardClass     int32
```
Comment (Mischanix) [inline pegasus/db.go]: I'd use ClassID instead of CardClass for the field name.  Card is implicit from the struct name, at least.

Comment (jleclanche) [inline pegasus/db.go]: Agreed; I was the one who recommended CardClass because it's what it's called in fireplace but that's more fitting.

**Comment by synap5e** [comment]
Oops, can you roll this back - I forgot to update 6b86c92 to use ClassID instead of CardClass

**Comment by synap5e** [comment]
OK, I've fixed synap5e:favorite-heroes now.

**Pull request merged**